### ISSUE TITLE
Session Storage as Option

### ISSUE THREAD

**Issue by britztopher** [sahat/satellizer#331]
Is there any way we can also use session storage to store token, as local storage persists to hard drive.  Im looking to just store until browser is closed.  Referring to #75 on this, but it looks as it got closed.  Any reason why?  I think just adding a `$authProvider.storage = 'session'` in $authProvider that sets the token in session object instead of  local storage.  Please let me know if anyone has added/working on this already.  If not, ill try to bang it out.

**Comment by sahat** [comment]
Yes it is possible. Now that localStorage calls have been refactored into the `storage` service, it should be fairly straightforward to add a configuration option that you mentioned above.

This should also resolve #247.

I will add it in the next version bump.

**Comment by britztopher** [comment]
@sahat  Awesome thanks

**Comment by heliodor** [comment]
It seems using "$auth.setStorage()" to switch away from the default does not get saved to where $auth.login() can use the newly set value. I'm trying to toggle the storage based on a 'remember me' flag before logging in. Is this a bug or am I doing something wrong?
